src/components/LeaderboardRanking.vue

```diff
@@ -77,27 +81,38 @@ export default {
   data() {
     return {
       cloudName: config.cloudName,
-    }
+    };
+  },
+  computed: {
+    ...mapGetters({
+      leaderboard: "leaderboards/currentLeaderboard",
+    }),
+    currentUser() {
+      const user = this.leaderboard.users?.find(
+        (user) => user?.userId === this.users[0].id
```

Comment:
Hmmm... I don't think there should be `null` objects inside the `users`? Did you run into any?
```suggestion
        (user) => user.userId === this.users[0].id
```

Reply:
Ah I just add optional chaining as second nature 😅, didnt encounter any errors though hehe